aspnetcore/security/index.md

```diff
@@ -43,6 +43,31 @@ ASP.NET Core and EF contain features that help you secure your apps and prevent
 
 There are more vulnerabilities that you should be aware of. For more information, see the other articles in the **Security and Identity** section of the table of contents.
 
+## Secure authentication flows
+
+We recommend using the most secure secure authentication option. For Microsoft services, the most secure authentication is [managed identities](/entra/identity/managed-identities-azure-resources/overview).
```

Comment:
Do you really mean *Microsoft* services here, or should it say *Azure* services?